gsa/src/web/pages/extras/feedstatuspage.js

```diff
@@ -265,4 +263,39 @@ const FeedStatus = () => {
   );
 };
 
-export default FeedStatus;
+FeedStatus.propTypes = {
+  feeds: PropTypes.array,
+};
+
+const FeedStatusWrapper = () => {
+  const gmp = useGmp();
+  const [feeds, setFeeds] = useState([]);
+
+  const loadFeeds = () =>
+    // cannot store gmp.feedstatus.readFeedInformation in separate variable
+    // causes undefined "this" in FeedStatus class
+    gmp.feedstatus.readFeedInformation().then(response => {
+      const {data} = response;
+      setFeeds(data);
+    });
+
+  const calculateSyncInterval = (feedsArray = []) => {
+    const hasFeed = hasValue(feedsArray[0]);
+
+    return hasFeed && hasValue(feedsArray[0].currently_syncing)
+      ? USE_DEFAULT_RELOAD_INTERVAL_ACTIVE
+      : USE_DEFAULT_RELOAD_INTERVAL;
+  }; // currently, if one feed is syncing then all feeds have the currently_syncing attribute if that changes, this will have to be redone
```

Comment:
Maybe we can just use something like `feedsArray.some(feed => feed.currently_syncing)` to check if at least one feed is updating. Then it should still work even if the behaviour changes.

Reply:
We can also do that. I was trying to avoid unnecessary looping but i guess with only 4 feeds it shouldn't be a problem. it's not like we're going to add an unlimited number of feeds later.